src/features/components/templates/banners/blade-shape-banner/index.tsx

```diff
@@ -108,40 +110,53 @@ const BladeShapeBanner = ({ data }: { data: BannerType }) => {
                                     <Localize translate_text={data.scan_box.heading_one} />
                                 </Typography.Paragraph>
                                 <Typography.Heading size={'xxs'} textcolor={'white'}>
-                                    <Localize translate_text={data.scan_box.heading_two} />
+                                    <Localize
+                                        translate_text={
+                                            is_appgallery_supported
+                                                ? data.scan_box.heading_two
+                                                : data.scan_box.heading_three
+                                        }
+                                    />
                                 </Typography.Heading>
                             </FlexBox.Box>
                         </FlexBox.Box>
                         <Flex.Box wrap={'wrap'} justify={'start'} pl={'12x'}>
-                            {data.os_apps.map((item) => (
-                                <Flex.Box
-                                    key={item.id}
-                                    align={'center'}
-                                    padding_block={'8x'}
-                                    basis={'6-12'}
-                                >
-                                    <Image
-                                        src={item.data.icon}
-                                        alt={item.data.text}
-                                        width={is_mobile ? 24 : 32}
-                                        height={is_mobile ? 24 : 32}
-                                    />
-                                    <Link pl={'3x'} url={item.data.url} no_hover>
-                                        {item?.data.smallText && (
-                                            <Typography.Paragraph textcolor={'white'} size={'xxs'}>
-                                                <Localize translate_text={item.data.smallText} />
+                            {data.os_apps
+                                .filter((item) => is_appgallery_supported || item.id !== 2)
```

Comment:
Why are we using `item.id !== 2` condition?

Reply:
Because we are selecting the Appgallery from the **os_apps** array. The id for the Appgallery in our codebase is 2.

 {
            id: 2,
            data: {
                icon: AppGalleryIcon,
                text: 'AppGallery',
                url: {
                    type: 'non-company',
                    href: deriv_go_huaweiappgallery_url,
                    target: '_blank',
                },
            },
        },